data/registry_model/test/test_registry_proxy_model.py

```diff
@@ -664,7 +664,7 @@ def test_renew_manifest_and_parent_tag_when_manifest_is_child_of_manifest_list(
         assert updated_tag.lifetime_end_ms > manifest_tag.lifetime_end_ms
         assert updated_list_tag.lifetime_end_ms > manifest_list_tag.lifetime_end_ms
 
-    def test_update_manifest_bytes_when_manifest_is_placeholder(
+    def test_update_manifest_bytes_and_layers_compressed_size_when_manifest_is_placeholder(
```

Comment:
is there a way to shorten this function name? This name by itself is over 80 characters which violates most style guides.

Reply:
done